### ISSUE TITLE
Anyalse and Check style - Appears not to be dealing with f strings

### ISSUE THREAD

**Issue by Phuket2** [zrzka/blackmamba#25]
BM version 1.0.2.  f strings from py 3.6 generates an error. I didn't copy the trace back as i think it will be pretty evident what the problem is to to you. 

```python
def my_handler(sender):
    print('in my sender {}'.format(sender.title))
    # the line below causes an error for control shift b
    print(f'in my sender {sender.title}') 
```

**Comment by zrzka** [comment]
Please, file an issue at Pythonista project. This is not Black Mamba issue. Black Mamba does use Pythonista bundled `pep8` (wrench - Check Style) and `pyflakes` (wrench - Analyze) packages. The only difference is that Black Mamba run both these tools and display results from both of them as annotations.

Try to run Analyze (wrench) your code and you'll get same result:

![6934ece9-23fe-4846-9685-64267703aed5](https://user-images.githubusercontent.com/1084172/31080666-a5f156fc-a789-11e7-9436-210c1f743068.png)

In other words, Pythonista bundled `pyflakes` doesn't support f strings.

**Comment by Phuket2** [comment]
oops, ok. Sorry about that.  I dont like to waste peoples time. It did seem strange that bm would cause that issue, but I think because I had just upgraded to the latest bm my brain stopped there. I will post on the Pythonista repo

**Comment by zrzka** [comment]
I already did it.

**Comment by zrzka** [comment]
Reopening because I replace pyflakes & pep8 with BM bundled flake8, which works.

**Comment by zrzka** [comment]
![90f813c9-4c80-4598-bbe3-43f004901d8f](https://user-images.githubusercontent.com/1084172/31237819-4b468260-a9f8-11e7-9b45-a80de6bd8124.png)

**Comment by zrzka** [comment]
Please, test 1.1.0 and reopen if it will not work for you.
